src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -76,7 +74,7 @@ function CurrencySearch({
     (currency: Currency) => {
       onCurrencySelect(currency)
       if (audioPlay) {
-        swapSound.play()
+        new Audio('swap.mp3').play()
```

Comment:
Is a good idea to lazy load the audio file, but creating new instance on each swap action may cause some overhead 🤔

Reply:
just a way to lazy the audio files。
you can destroy the instance after play，in another way，you should to analysis how many times does caker to swap on each visit，overhead should base on the data as you know。for me，the load of the audio is useless due to my habit。